packages/synapse-react-client/src/components/TableColumnSchemaEditor/ColumnModelForm.tsx

```diff
@@ -123,140 +142,176 @@ export default function ColumnModelForm(props: ColumnOrSubcolumnFormProps) {
       </Box>
       <Box
         sx={{
+          my: 'auto',
           gridColumn: isJsonSubColumn
             ? /* If this is a JSON Subcolumn, we reduce the width of this grid column to create space to render the visual hierarchical line */
               '3 / span 1'
             : /* Otherwise, span across both grid columns  */
               '2 / span 2',
         }}
       >
-        <TextField
-          value={columnModel.name}
-          placeholder={isJsonSubColumn ? 'Facet name' : 'Column name'}
-          onChange={e => {
-            dispatch({
-              type: 'setColumnModelValue',
-              columnModelIndex,
-              jsonSubColumnModelIndex: jsonSubColumnIndex,
-              value: {
-                ...columnModel,
-                name: e.target.value,
-              },
-            })
-          }}
-          InputProps={{
-            sx: fieldSx,
-            inputProps: {
-              'aria-label': 'Name',
-            },
-          }}
-          fullWidth
-        />
-      </Box>
-      <Box>
-        <FormControl fullWidth>
-          <Select
-            label="Column Type"
-            value={columnModel.columnType}
+        {isDefaultColumn ? (
```

Comment:
Most of these changes are adding ternary expressions to show the value instead of an input if it's a default column